### PR TITLE
Defender Plasma Costs Doubled

### PR DESCRIPTION
**Pull request by Surrealaser**
Per title.

Also redefined Defender related variables to constants as appropriate.

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -492,9 +492,12 @@ var/list/global_mutations = list() // list of hidden mutation things
 
 //defender defines
 
-#define DEFENDER_HEADBUTT_COST 20
-#define DEFENDER_TAILSWIPE_COST 30
-
+#define DEFENDER_HEADBUTT_COST			20
+#define DEFENDER_HEADBUTT_COOLDOWN		40
```
Comment (Rohesie) [inline code/__DEFINES/mobs.dm]:
```suggestion
#define DEFENDER_HEADBUTT_COOLDOWN		4 SECONDS
```

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -492,9 +492,12 @@ var/list/global_mutations = list() // list of hidden mutation things
...
+#define DEFENDER_TAILSWIPE_COST			30
+#define DEFENDER_TAILSWIPE_COOLDOWN		120
```
Comment (Rohesie) [inline code/__DEFINES/mobs.dm]:
```suggestion
#define DEFENDER_TAILSWIPE_COOLDOWN		12 SECONDS
```

File: code/__DEFINES/mobs.dm [hunk 3]
```diff
@@ -492,9 +492,12 @@ var/list/global_mutations = list() // list of hidden mutation things
...
+#define DEFENDER_FORTIFY_COOLDOWN		10
```
Comment (Rohesie) [inline code/__DEFINES/mobs.dm]:
```suggestion
#define DEFENDER_FORTIFY_COOLDOWN		1 SECOND
```

File: code/__DEFINES/mobs.dm [hunk 4]
```diff
@@ -492,9 +492,12 @@ var/list/global_mutations = list() // list of hidden mutation things
...
+#define DEFENDER_CREST_DEFENSE_COOLDOWN	10
```
Comment (Rohesie) [inline code/__DEFINES/mobs.dm]:
```suggestion
#define DEFENDER_CREST_DEFENSE_COOLDOWN	1 SECOND
```

**Comment by spookydonut** [comment]
Merge this and I'll resolve the conflicts in #695

**Pull request merged**